ex3/index.js

```diff
@@ -0,0 +1,15 @@
+function letterCount(sir, litera) {
+    let sirLower = sir.toLowerCase();
+    let literaLower = litera.toLowerCase();
+    let count = 0;
+    for (let i = 0; i < sirLower.length; i++) {
+    if (sirLower[i] === literaLower) {
```

Comment:
lipseste identarea 
de fiecare dat cand deschidem acolada `{` trebuie sa lasam un tab (4 spatii) in plus la urmatorul rand.